Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby "3.3.0"
+ruby File.read(".ruby-version").strip
```

Comment:
Would `ruby file: ".ruby-version"` work?

Reply:
It would! Have just updated.